src/pages/Missions.jsx

```diff
@@ -19,6 +23,21 @@ const missionsArray = [
 ];
 
 const Missions = () => {
+
+  const dispatch = useDispatch();
+  const missions = useSelector(state => state.missions);
+
+  useEffect(() => {
```

Comment:
Nice Job Buddy, I have selected here multiple lines, but would be great if you could move the logic inside **useEffet**, to the **App.jsx** component, this will help to make sure that the full website **only fetches your data the first time the website is loaded** in the browser, if you keep this useEffect here, it will fetch your data every time you change to a different section within the website, and would not allow you to show changes in the `my profile`.